.github/tests-visual/visual-record.js

```diff
@@ -123,8 +123,8 @@ function countDiffPixels(img1path, img2path ) {
 async function processScreenshot(page, screenshotName) {
     mkdirp.sync(WORKING_DIR);
 
-    // hide blinking cursor
-    await page.addStyleTag({ content: 'input { caret-color: transparent !important; }' });
+    // hide blinking cursor/icon
+    await page.addStyleTag({ content: 'input { caret-color: transparent !important; } .rex-pulse { animation: initial }' });
```

Comment:
Macht es sinn, die animations für alle nodes zu deaktivieren?

```suggestion
    await page.addStyleTag({ content: 'input { caret-color: transparent !important; } * { animation: initial !important}' });
```